app/views/users/getting_started.haml

```diff
@@ -4,8 +4,13 @@
       .hero-unit
         %h1.text-center
           = t('.well_hello_there')
+
         %h3.text-center
           = t(".community_welcome")
+          %h3.text-center
+            = link_to  "Get started with diaspora* tutorials!", "https://diasporafoundation.org/tutorials"
```

Comment:
I also had a look at that issue and wasn't sure which sentence we should put. I had in mind something like "To get off to a good start, you can start by reading [the tutorials]". @goobertron what do you think?

Reply:
I think 'Get started with diaspora\* tutorials!' is all right. Or it could be 'Check out our tutorials!'

Other suggestions:
- 'Get off to a good start by reading our tutorials!'
- 'Read our tutorials to help you get started.'
- 'Let our tutorials help you use diaspora\*.'

Anything like that is fine with me.

But the string, and the title, need to be translatable. And 'Diaspora\*' in the title should be lower-case.

Thanks for asking me.

Reply:
>Get off to a good start by reading our tutorials!

This one is nice ;) @kyle-fitz  can you please change the sentence to that and make it translatable?

Reply:
Is there a link to a guide on how to make text strings translatable, in case @kyle-fitz doesn't know about this?